tests/fileformats/top/test_TOP.py

```diff
@@ -0,0 +1,67 @@
+import gromacs as gmx
+from unittest.mock import patch, mock_open
```

Comment:
Can you use pytest's mocking support https://docs.pytest.org/en/7.1.x/how-to/monkeypatch.html so that it stays within pytest? Alternatively, we could install [pytest-mock](https://pypi.org/project/pytest-mock/).

Reply:
ah, what's the motivation here – is it just consistency? I really, really don't enjoy how monkeypatch solves the problem. unittest is mock is basically a built-in and I see it used more frequently. 

Happy to add more tests – that's always a good idea – but i will need some direction as to what to cover? My tests already covered all the possible fu values.

Reply:
consistency  to stay within one testing framework

Reply:
test coverage itself looks decent enough